File: mvvm/src/main/java/com/yanghao277dev/mvvmbase/http/HttpLogger.java
package com.yanghao277dev.mvvmbase.http;


import com.yanghao277dev.mvvmbase.base.MyLog;

import okhttp3.logging.HttpLoggingInterceptor;

public class HttpLogger implements HttpLoggingInterceptor.Logger {
    private static String TAG = HttpLogger.class.getSimpleName();

    @Override
    public void log(String message) {
        MyLog.INSTANCE.netState(TAG,message);
    }
}

File: mvvm/src/main/java/com/yanghao277dev/mvvmbase/http/HttpLogger.kt
package com.yanghao277dev.mvvmbase.http

import com.yanghao277dev.mvvmbase.base.MyLog.netState
import okhttp3.logging.HttpLoggingInterceptor

class HttpLogger : HttpLoggingInterceptor.Logger {
    override fun log(message: String) {
        netState(TAG, message)
    }

    companion object {
        private val TAG = HttpLogger::class.java.simpleName
    }
}